BearlyHumans/squirrelville
# Island Design Changes

Merging major design changes into main.

All of the design changes affect only Luke and Toby's scenes. However, there will be minor changes to the player package, more specifically, the move and climb inspector values.

This request needs to be accepted so that I can tag the **Respawn Zones** correctly.

To Test:

- Load into **ISLAND Luke** scene.

- Play the game and make sure everything is working

- Talk to the **Koala** to make sure the dialogue works

- Check that the peak climb looks like it does in the **image** below
![image](https://user-images.githubusercontent.com/72909556/135652186-a230957d-940a-4756-82e1-6186e25b21a8.png)

- Check that **RespawnZones** are placed throughout the world. These can be found at the bottom of the Hierarchy (these will not be working yet)
